core/storage/base_model/gae_models.py

```diff
@@ -224,16 +262,23 @@ def _require_not_marked_deleted(self):
             raise Exception('This model instance has been deleted.')
 
     def _compute_snapshot(self):
-        """Generates a snapshot (a Python dict) from the model fields."""
+        """Generates a snapshot (dict) from the model property values."""
         return self.to_dict(exclude=['created_on', 'last_updated'])
 
     def _reconstitute(self, snapshot_dict):
-        """Makes this instance into a reconstitution of the given snapshot."""
         self.populate(**snapshot_dict)
         return self
 
     def _reconstitute_from_snapshot_id(self, snapshot_id):
-        """Makes this instance into a reconstitution of the given snapshot."""
+        """Gets instance in a reconstituted state according to the given
```

Comment:
Gets a reconstituted instance of this model class, based on the given snapshot id.